{
  "0 debug pnpm:scope": {
    "selected": 1
  },
  "1 info pnpm": {
    "err": {
      "name": "Error",
      "message": "not found: dev",
      "code": "ENOENT",
      "stack": "Error: not found: dev\n    at getNotFoundError (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41700:51)\n    at Function.whichSync [as sync] (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41777:13)\n    at getCommandAbsolutePathSync (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43342:44)\n    at default_1 (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43351:32)\n    at /usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:179510:39"
    }
  },
  "2 error pnpm": {
    "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
    "prefix": "/Users/idiotic/Works/my-git/reject996-api",
    "err": {
      "name": "pnpm",
      "message": "not found: dev",
      "code": "ERR_PNPM_RECURSIVE_EXEC_FIRST_FAIL",
      "stack": "Error: not found: dev\n    at getNotFoundError (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41700:51)\n    at Function.whichSync [as sync] (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:41777:13)\n    at getCommandAbsolutePathSync (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43342:44)\n    at default_1 (/usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:43351:32)\n    at /usr/local/lib/node_modules/pnpm/dist/pnpm.cjs:179510:39"
    }
  }
}